src/pages/cardsPage/CardsPage.jsx

```diff
@@ -0,0 +1,59 @@
+import { React, useState, useEffect } from 'react';
```

Comment:
Rename the page with uppercase letter. If we have a component or page - it should be capitalized